src/cpp/include/openvino/genai/parsers.hpp

```diff
@@ -222,7 +222,7 @@ class OPENVINO_GENAI_EXPORTS ReasoningIncrementalParser : public IncrementalPars
      * @return std::string Filtered text with reasoning content processed according to configuration
```

Comment:
Update doc.  Replace message -> delta_message in `@param message JsonContainer to store parsed results and reasoning metadata`